.vscode/launch.json

```diff
@@ -8,120 +8,51 @@
+            "program": "${workspaceFolder}/examples/${input:example}/src/main.py",
             "console": "integratedTerminal",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "native",
-                "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/dynamic-rule (native)",
-            "cwd": "${workspaceFolder}/examples/dynamic-rule",
-            "program": "${workspaceFolder}/examples/dynamic-rule/src/main.py",
-            "console": "integratedTerminal",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "native",
-                "SDV_MQTT_ADDRESS": "mqtt://localhost:1883",
-                "SDV_VEHICLEDATABROKER_ADDRESS": "grpc://localhost:55555"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/datapoint-set (dapr)",
-            "cwd": "${workspaceFolder}/examples/datapoint-set",
-            "program": "${workspaceFolder}/examples/datapoint-set/src/main.py",
-            "console": "integratedTerminal",
-            "preLaunchTask": "dapr-SeatAdjuster-run",
-            "postDebugTask": "dapr-SeatAdjuster-stop",
-            "env": {
-                "SDV_MIDDLEWARE_TYPE": "dapr",
-                "DAPR_GRPC_PORT": "50001",
-                "DAPR_HTTP_PORT": "3500",
-                "SEATSERVICE_DAPR_APP_ID": "seatservice",
-                "VEHICLEDATABROKER_DAPR_APP_ID": "vehicledatabroker"
-            }
-        },
-        {
-            "type": "python",
-            "request": "launch",
-            "name": "examples/atomic-set (dapr)",
-            "cwd": "${workspaceFolder}/examples/atomic-set",
-            "program": "${workspaceFolder}/examples/atomic-set/src/main.py",
-            "console": "integratedTerminal",
-            "preLaunchTask": "dapr-SeatAdjuster-run",
-            "postDebugTask": "dapr-SeatAdjuster-stop",
+            "preLaunchTask": "dapr-VehicleApp-run",
+            "postDebugTask": "dapr-VehicleApp-stop",
             "env": {
                 "SDV_MIDDLEWARE_TYPE": "dapr",
                 "DAPR_GRPC_PORT": "50001",
                 "DAPR_HTTP_PORT": "3500",
+                "HVACSERVICE_DAPR_APP_ID": "hvacservice",
                 "SEATSERVICE_DAPR_APP_ID": "seatservice",
                 "VEHICLEDATABROKER_DAPR_APP_ID": "vehicledatabroker"
             }
         }
+    ],
+    "inputs": [
+        {
+            "description": "Select an example to launch",
+            "id": "example",
+            "type": "pickString",
+            "options": [
+                "array-datatype",
+                "atomic-set",
+                "datapoint-set",
+                "dog-mode",
+                "dynamic-rule",
+                "static-rule",
```

Comment:
I noticed seat-adjuster is missing here. Is this on purpose?

Reply:
Yes, because it won't run here as the other examples - but would be importable as template app.
Problem is, that models aren't compatible at the moment.

Reply:
As discussed with Dominc: Should be importable to template.